examples/typescript/twisted_el_gamal.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-console */
+
+/**
+ * This example shows how a balance can be encrypted and decrypted using the Twisted ElGamal
+ */
+
+import { TwistedElGamal, TwistedEd25519PrivateKey } from "@aptos-labs/ts-sdk";
```

Comment:
update this import

Reply:
examples has been removed, test files enough to be used as examples